scripts/install-chef-suse.sh

```diff
@@ -911,7 +911,7 @@ fi
 
 # Use current time zone
 (
-    . /etc/sysconfig/clock || : Ignoring missing /etc/sysconfig/clock
+    TIMEZONE=$(LC_ALL=C timedatectl status | grep "zone:" | cut -d: -f2 | cut -d' ' -f2)
```

Comment:
Alternatively you could sed this: `timedatectl status | sed -e "/zone:/ s/^.*: \([^ ]\+\).*/\1/; t; d"`...